Framework/DataHandling/src/LoadNexusProcessed.cpp

```diff
@@ -421,9 +424,10 @@ void LoadNexusProcessed::exec() {
     std::string base_name = getPropertyValue("OutputWorkspace");
     // First member of group should be the group itself, for some reason!
 
-    // load names of each of the workspaces and check for a common stem
+    // load names of each of the workspaces. Note that if we have duplicate
+    // names then we don't select them
     std::vector<std::string> names(nWorkspaceEntries + 1);
-    bool commonStem = bIsMultiPeriod || checkForCommonNameStem(root, names);
+    extractWorkspaceNames(root, names);
```

Comment:
I know this is picky but it would be good to gradually cleanup functions/methods that use this in/out style and prefer something that just returns the new vector.

With move semantics and return value optimization having `names = extractWorkspaceNames(root);` is much cleaner.